### PR TITLE
Update before_you_start.mdx

### PR DESCRIPTION
**Pull request by ftouserkani-edb**
The link provided in the document needs to be updated.

### CODE REVIEW

File: product_docs/docs/postgres_for_kubernetes/1/before_you_start.mdx [hunk 1]
```diff
@@ -76,7 +76,7 @@ specific to Kubernetes and PostgreSQL.
 : `kubectl` is the command-line tool used to manage a Kubernetes cluster.
 
 EDB Postgres for Kubernetes requires a Kubernetes version supported by the community. Please refer to the
-["Supported releases"](/resources/platform-compatibility#pgk8s) page for details.
+["Supported releases"](https://cloudnative-pg.io/documentation/current/supported_releases/) page for details.
```
Comment (djw-m) [inline product_docs/docs/postgres_for_kubernetes/1/before_you_start.mdx:79]:
```suggestion
["Supported releases"](https://www.enterprisedb.com/resources/platform-compatibility#pgk8s) page for details.
```

**Review by djw-m — COMMENTED** [review]
The correction should be to point to the platform compatibility page, not the external supported releases page which doesn't say anything about EDB's support commitments.

**Review by gvasquezvargas — APPROVED** [review]
LGTM! Is pointing at the right page now.

File: product_docs/docs/postgres_for_kubernetes/1/before_you_start.mdx [hunk 2]
```diff
@@ -76,7 +76,7 @@ specific to Kubernetes and PostgreSQL.
 : `kubectl` is the command-line tool used to manage a Kubernetes cluster.
 
 EDB Postgres for Kubernetes requires a Kubernetes version supported by the community. Please refer to the
-["Supported releases"](/resources/platform-compatibility#pgk8s) page for details.
+["Supported releases"](https://www.enterprisedb.com/resources/platform-compatibility#pgk8s) page for details.
```
Comment (gvasquezvargas) [inline product_docs/docs/postgres_for_kubernetes/1/before_you_start.mdx:79]: Just a nit, I don't think we need the "" here: 
```suggestion
[Supported releases](https://www.enterprisedb.com/resources/platform-compatibility#pgk8s) page for details.
```

**Pull request merged**